### DISCUSSION TITLE
Nested template instances inherit the promoted labels of parent template

### DISCUSSION THREAD

**Discussion by darkpoetry** [TriliumNext/Trilium#3445] [General]
Not sure if this is a bug or just doing it wrong. I'm an attorney (former software dev) working on a way to organize my legal cases, so I tried creating two templates. One is "defendant" and the other is "court case". 

Defendant:
`#template #iconClass="bx bxs-cool" #label:Age=promoted,single,number`

Court case:
`#template #sorted #label:FiledOn=promoted,single,date #iconClass="bx bx-cabinet" `

The defendant template has a ~child:template=[Court Case] entry, so that any new notes created below follow that template. 

The problem is that the [Court Case] instances show the promoted "Age" label in addition to its own "FiledOn" even though inherit is not selected.

**Comment by zadam** [comment]
Hello, your setup sounds to me correct, I tried to reproduce the beavior you describe, but it behaves correctly:

<img width="258" alt="image" src="https://user-images.githubusercontent.com/617641/208941359-642b5c6c-ed80-42c8-a455-61f338593c6c.png">

<img width="588" alt="image" src="https://user-images.githubusercontent.com/617641/208941710-de8a26e9-660f-40e5-ad85-94eff3ee2e06.png">

<img width="660" alt="image" src="https://user-images.githubusercontent.com/617641/208941818-8a6f89c8-e118-4cdf-b9b8-8cb3b976f55b.png">

<img width="586" alt="image" src="https://user-images.githubusercontent.com/617641/208941922-ed04b05f-1e6d-4ee8-86b5-705ceb0e421e.png">

<img width="550" alt="image" src="https://user-images.githubusercontent.com/617641/208941999-004d7b8e-6d6e-4c5b-a2b7-f7674e81f0d3.png">

**Comment by darkpoetry** [comment]
I upgraded to 57.5.5 and the issue is gone. Appreciate the help. This is the greatest app ever for my continually changing information maps.
